mfc: OK > (venv) Entered the Python 3.12.4 virtual environment (>= 3.8).

      .=++*:          -+*+=.        | bok@bok [Linux]
     :+   -*-        ==   =* .      | ---------------
   :*+      ==      ++    .+-       | 
  :*##-.....:*+   .#%+++=--+=:::.   | --jobs 8
  -=-++-======#=--**+++==+*++=::-:. | --no-mpi --no-gpu --no-debug --no-gcov --no-unified
 .:++=----------====+*= ==..:%..... | 
  .:-=++++===--==+=-+=   +.  :=     | 
  +#=::::::::=%=. -+:    =+   *:    | ----------------------------------------------------------
 .*=-=*=..    :=+*+:      -...--    | $ ./mfc.sh (build, run, test, clean, count, packer) --help

 

Error: Input file /home/bok/dev/MFC/examples/2D_bubbly_steady_shock/case.py: data must not contain {'adv_alphan', 'ib_wrt'} properties


mfc: ERROR > main.py finished with a 143 exit code.
mfc: (venv) Exiting the Python virtual environment.
